src/ui/pages/SendTransaction/TransactionConfiguration/useTransactionFee.ts

```diff
@@ -195,30 +187,26 @@ function calculateTransactionCosts({
   };
 }
 
-const findMatchingEip1559 = (
-  eip1559GasPrices: EIP1559GasPrices,
-  eip1559: EIP1559
-) => {
+const findMatchingEip1559 = (gasPrices: ChainGasPrice, eip1559: EIP1559) => {
   const isMatch = (value1: EIP1559, value2: EIP1559) =>
-    value1.max_fee === value2.max_fee &&
-    value1.priority_fee === value2.priority_fee;
-  const { rapid, fast, standard, slow } = eip1559GasPrices;
-  return [rapid, fast, standard, slow]
+    value1.maxFee === value2.maxFee &&
+    value1.priorityFee === value2.priorityFee;
+  const { fast, average } = gasPrices;
+  return [fast, average]
     .filter(isTruthy)
-    .find((eip1559Info) => isMatch(eip1559Info, eip1559));
+    .find(
+      ({ eip1559: eip1559Info }) => eip1559Info && isMatch(eip1559Info, eip1559)
+    );
 };
 
 function getFeeTime(
   gasPrice: GasPriceObject | null,
   chainGasPrices: ChainGasPrice | null
 ) {
-  if (gasPrice?.eip1559 && chainGasPrices?.info.eip1559) {
+  if (gasPrice?.eip1559 && chainGasPrices?.fast.eip1559) {
```

Comment:
we're not checking "average" here, not sure if critical

Reply:
Should not be critical, cause fast option is a default one. Good point anyway